MathLib/mechanics_invariants.h

```diff
@@ -33,7 +33,7 @@ struct Invariants final
     static Eigen::Matrix<double, KelvinVectorSize, KelvinVectorSize> const
         P_sph;
     /// Kelvin mapping of 2nd order identity tensor.
-    static Eigen::Matrix<double, KelvinVectorSize, 1> const identity2;
+    static Eigen::Matrix<double, KelvinVectorSize, 1> const Identity2;
```

Comment:
If the capital `I` shall stress the constness, I'd suggest to have it all-caps. Otherwise I'd prefer to have data members lowercase and only types start with an uppercase letter.